Drop deprecation header from “Synchronous and asynchronous requests” doc (#21038)
mdn/content: files/en-us/web/api/xmlhttprequest/synchronous_and_asynchronous_requests/index.md

## Synchronous request

> Synchronous XHR requests often cause hangs on the web. But developers typically don't notice the problem because the hang only manifests with poor network conditions or when the remote server is slow to respond. Synchronous XHR is now in deprecation state. The recommendation is that developers move away from the synchronous API and instead use asynchronous requests.

Reviewer: I still think this is better in a warning box (we can make it shorter if necessary). Not only for reader attention, but also because it looks a bit detached from the rest of the content (it's much more normative than descriptive).

Author: > I still think this is better in a warning box (we can make it shorter if necessary). Not only for reader attention, but also because it looks a bit detached from the rest of the content (it's much more normative than descriptive).

OK — let’s go with that (just now pushed an update)

## Synchronous request

> > **Warning:** Synchronous XHR requests often cause hangs on the web. But developers typically don't notice the problem because the hang only manifests with poor network conditions or when the remote server is slow to respond. Synchronous XHR is now in deprecation state. The recommendation is that developers move away from the synchronous API and instead use asynchronous requests.

Reviewer: ```suggestion
> **Warning:** Synchronous XHR requests often cause hangs on the web, especially with poor network conditions or when the remote server is slow to respond. Synchronous XHR is now deprecated and should be avoided in favor of asynchronous requests.
```
